### PR TITLE
Updates bosh-env doc to show bosh-env usage

### PR DESCRIPTION
**Pull request by clhynfield**
Previously it showed usage for apply-changes

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@clhynfield Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cf/om&pullRequestId=339)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=pivotal-cf/om&pullRequestId=339) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@clhynfield Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cf/om&pullRequestId=339)!

**Comment by clhynfield** [comment]
Sorry, should have noted that this fixes #322

File: docs/bosh-env/README.md [hunk 1]
```diff
@@ -9,21 +9,19 @@ The `bosh-env` command setup environment variables to target bosh director and/o
...
+  --help, -h        bool    prints this usage information (default: false)  --password, -p, OM_PASSWORD
+        string  admin password for the Ops Manager VM (not required for unauthenticated commands)  --request-timeout, -r, OM_REQUEST_TIMEOUT        int     timeout in seconds for HTTP requests to Ops Manager (default: 1800)  --skip-ssl-validation, -k, OM_SKIP_SSL_VALIDATION      bool    skip ssl certificate validation during http requests (default: false)
+  --target, -t, OM_TARGET        string  location of the Ops Manager VM  --trace, -tr, OM_TRACE        bool    prints HTTP requests and response payloads  --username, -u, OM_USERNAME
+        string  admin username for the Ops Manager VM (not required for unauthenticated commands)
+  --version, -v        bool    prints the om release version (default: false)Command Arguments:  --shell-type           string  Prints for the given shell (posix|powershell)  --ssh-private-key, -i  string  Location of ssh
```
Comment (kcboyle) [inline docs/bosh-env/README.md]: Should Command Arguments be on a new line?

Comment (clhynfield) [reply docs/bosh-env/README.md]: Yep, sorry – I was trying to get this in quickly, didn’t proofread. I’ll push a clean-up commit shortly

**Comment by kcboyle** [comment]
Looks good! Thanks @clhynfield !

**Pull request merged**